dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerTest.java

```diff
@@ -469,8 +470,9 @@ void getTrackedEntityByIdyWithFieldsRelationshipsNoAccessToTrackedEntityType() {
     relationship(from, to);
     this.switchContextToUser(user);
 
-    GET("/tracker/trackedEntities/{id}?fields=relationships", from.getUid())
```

Comment:
Does this style not work? or does the `assertEquals()` version give better error messages